relay-server/src/actors/processor.rs

```diff
@@ -2733,27 +2743,63 @@ impl EnvelopeProcessorService {
             }
         }
     }
+
+    async fn subscribe_global_config(&self) -> Receiver<Arc<GlobalConfig>> {
+        match self.inner.global_config.send(Subscribe).await {
+            Ok(c) => c,
+            Err(e) => {
+                relay_log::error!(
+                    error = &e as &dyn Error,
+                    "failed to subscribe to GlobalConfigService",
+                );
+                std::process::exit(1);
+            }
+        }
+    }
+
+    async fn get_global_config(&self) -> Arc<GlobalConfig> {
+        match self.inner.global_config.send(Get).await {
+            Ok(global_config) => global_config,
+            Err(e) => {
+                relay_log::error!(error = &e as &dyn Error, "failed to fetch global config");
+                std::process::exit(1);
+            }
+        }
+    }
```

Comment:
@iker-barriocanal I refactored this piece of code you wrote, using join! now since the two requests shouldn't depend on each other.  
 
it would be nice to get the global config in the constructor, but I suppose that's not done because it would be an async operation, is that right?